morse_code.rb

```diff
@@ -0,0 +1,22 @@
+def morse_code(code)
+  codes = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V',
+    '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z'
+  }
+  codes[code]
+end
+
+def full_decode(full_encryption)
+  full_encryption = full_encryption.split('   ')
+  full_encryption.map { |word| decode(word) }.join(' ')
+end
+
+def decode(encryption)
+  letters = encryption.split
+  letters.map { |letter| morse_code(letter) }.join
+end
```

Comment:
- [SUGGESTION] In line with the above concern, this one could be named `decode_word` based on its purpose.